onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -85,10 +85,11 @@ namespace perftest {
           test_config.machine_config.provider_type_name = onnxruntime::kDnnlExecutionProvider;
         } else if (!CompareCString(optarg, ORT_TSTR("ngraph"))) {
           test_config.machine_config.provider_type_name = onnxruntime::kNGraphExecutionProvider;
-        } else if (!CompareCString(optarg, ORT_TSTR("tensorrt"))) {
-          test_config.machine_config.provider_type_name = onnxruntime::kTensorrtExecutionProvider;
         } else if (!CompareCString(optarg, ORT_TSTR("openvino"))) {
           test_config.machine_config.provider_type_name = onnxruntime::kOpenVINOExecutionProvider;
+          test_config.run_config.optimization_level = ORT_DISABLE_ALL;
```

Comment:
if this is required/recommended, this should be noted in the OpenVINO EP documentation somewhere.
otherwise, it may impact how much of the graph can execute on OPENVINO EP.

Reply:
Added recommendation and instructions in OpenVINO EP documentation